docs/openshift.asciidoc

```diff
@@ -29,9 +29,19 @@ Before deploying an Elasticsearch cluster with ECK, make sure you correctly appl
 [float]
 [id="{p}-openshift-deploy-the-operator"]
 === Deploy the operator
+
 . Apply the all-in-one template, as described in the link:k8s-quickstart.html[quickstart].
 +
-[source,shell,subs="attributes"]
+If you are using a Kubernetes cluster in version 1.12 or below, use:
++
+[source,sh,subs="attributes"]
+----
+oc apply -f https://download.elastic.co/downloads/eck/{eck_version}/all-in-one-no-validation.yaml
+----
++
+If you are using a Kubernetes cluster from version 1.13, use:
```

Comment:
We should not use K8S version in an Openshift documentation.
IIRC:
* OpenShit 3.11 → K8S 1.1
* OpenShit 4.0 → K8S 1.12
* OpenShit 4.1 → K8S 1.13
* OpenShit 4.2 → K8S 1.14